src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -50,7 +51,11 @@ export class BucketDetailComponent implements OnInit {
   showCreateFolder = false;
   createFolderForm:FormGroup;
   showErrorMsg = false;
-  UPLOAD_UPPER_LIMIT = 1024*1024*1024*2;
+  isReadyCopy = true;
```

Comment:
This looks like code for CopyObject. Please check.

Reply:
Yes, you are right. I will pull the latest code of delovement and resubmit the code